data/mods/Magiclysm/items/alchemy_items.json

```
@@ -41,6 +41,15 @@
     "name_plural": "copper infusion bracelets",
     "description": "This bracelet has runes engraved on it. You sense a faint air of mysticism when you look at it. It would be useful for imbuing mana into material."
   },
+  {
+    "id": "silver_infuser",
+    "copy-from": "silver_bracelet",
+    "type": "ARMOR",
+    "qualities": [ [ "MANA_INFUSE", 2 ] ],
+    "name": "silver infusion bracelet",
+    "name_plural": "silver infusion bracelets",
+    "description": "This bracelet has runes engraved on it. You sense a faint air of mysticism when you look at it. It would be useful for imbuing mana into material. Place-Holder"
```

```suggestion
    "description": "This bracelet has runes engraved on it. You sense a faint air of mysticism when you look at it. It would be useful for imbuing mana into material."
```
no need to tell players this. plus i don't have a problem with the silver one actually being the level 2 mana infusing device, even if we decide to move the recipe to another book.